### PR TITLE
BXMSPROD-560 Revert "Remove yarn.lock (#20)" yarn.lock entry removed from .gitignore

### PR DESCRIPTION
**Pull request by Ginxo**
BXMSPROD-560

### CODE REVIEW

**Comment by tiagobento** [comment]
@Ginxo hi! Following up on your comment on the previous PR.. What kind of errors did you see on `appformer-js`? Do you have some logs or a JIRA somewhere?

**Comment by domhanak** [comment]
@ginxo I do not see the tool being installed or executed in the Jenkins build logs. Imho you need to install it in Prepare part (globally) and then execute it before `yarn run init ...` step in build.

**Comment by Ginxo** [comment]
@tiagobento as I said here https://github.com/kiegroup/kogito-tooling/pull/29#issuecomment-548227537 there are no current problems with appformer-js specifically but they can appear in the future (because the lock file absence), so we need to keep the version integrity somehow and locking files is the way

File: Jenkinsfile [hunk 1]
```diff
@@ -22,6 +22,8 @@ pipeline {
         }
         stage('Prepare') {
             steps {
+                sh "npm install lock-treatment-tool --global-style --no-package-lock --no-save --registry=${NPM_REGISTRY_URL}"
```
Comment (Ginxo) [inline Jenkinsfile]: @domhanak I saw the error, but... is it the proper place?

Comment (domhanak) [reply Jenkinsfile]: @Ginxo I believe it is, add `-g` after npm install and we'll see if it works

Comment (Ginxo) [reply Jenkinsfile]: @domhanak command modified to be installed globally

**Comment by domhanak** [comment]
@Ginxo I am not sure why, but I can't see the steps you added in logs.

**Comment by Ginxo** [comment]
@domhanak I noticed that, is there a way to relaunch it (without adding anything new)?

**Comment by domhanak** [comment]
Jenkins retest this

**Comment by domhanak** [comment]
@Ginxo not sure we will see if the above comment works

**Comment by domhanak** [comment]
@Ginxo check here - https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/kogito/job/kogito-tooling/job/PR-32/4/console , I modified the steps a bit:
I kept ` sh "npm install -g lock-treatment-tool --registry=${NPM_REGISTRY_URL}"` in prepare step
but I moved
`locktt --registry=url` before `sh("yarn run init & yarn run build:prod")`

could you please update it?

**Comment by domhanak** [comment]
Rerunning it again to double check

**Comment by Ginxo** [comment]
@domhanak done

**Comment by domhanak** [comment]
@Ginxo Thanks, I will manually add your changes and do a rerun in case the build does not account for them. But this should be all! Thanks! :)

**Review by domhanak — APPROVED** [review]
Ready for merging

**Comment by tiagobento** [comment]
@domhanak Jenkins is still failing. Is there anything missing?

**Comment by tiagobento** [comment]
@Ginxo Please make sure to update the yarn.lock file, since `master` has been updated.

**Comment by Ginxo** [comment]
@tiagobento yarn.lock updated according to the latest master version

**Comment by tiagobento** [comment]
Well, this looks good to me. I don't understand why Jenkins is still failing, though. @domhanak @mareknovotny @Ginxo It should start working for future PRs once merged?

**Comment by Ginxo** [comment]
@tiagobento according to https://github.com/kiegroup/kogito-tooling/pull/36 yarn.lock removed

**Comment by domhanak** [comment]
@tiagobento It fails because the Jenkinsfile change is not taken into account. I rerun it with the changes from this PR and it passes. So feel free to merge.

**Review by ederign — APPROVED** [review]

**Pull request merged**